PEP 757: address gpshead's review (#4144)
python/peps: peps/pep-0757.rst

   Create a :c:type:`PyLongWriter`.

   On success, allocate *\*digits* and return a writer.
   On error, set an exception and return ``NULL``.

   *negative* is ``1`` if the number is negative, or ``0`` otherwise.

   *ndigits* is the number of digits in the *digits* array.  It must be
   greater than 0.

   The caller can either initialize the array of digits *digits* and then
   either call :c:func:`PyLongWriter_Finish` to get a Python :class:`int` or
   :c:func:`PyLongWriter_Discard` to destroy the writer instance.  Digits must
   be in the range [``0``; ``(1 << bits_per_digit) - 1``]  (where the
   :c:struct:`~PyLongLayout.bits_per_digit` is the number of bits per digit).

>    Unused most significand digits must be set to ``0``.

Reviewer: ```suggestion
   The unused most-significant digits must be set to ``0``.
```

   The function takes care of normalizing the digits and converts the
   object to a compact integer if needed.

>    The writter instance is invalid after the call.

Reviewer: ```suggestion
   The writer instance is invalid after the call.
```

.. c:function:: void PyLongWriter_Discard(PyLongWriter *writer)

   Discard a :c:type:`PyLongWriter` created by :c:func:`PyLongWriter_Create`.

>    The writter instance is invalid after the call.

Reviewer: ```suggestion
   The writer instance is invalid after the call.
```
